### PR TITLE
Inclusive Naming: repo links, comments, couple of test files, 

### PR DESCRIPTION
**Pull request by rohit-nayak-ps**
## Description

Based on results from `git grep -i master`, changes references to master in:
* repo links back to github master branch to main branch
* most references in comments
* a few functions/variables in a couple of test files

## Checklist
- [ ] Should this PR be backported? - No
- [x] Tests were added or are not required
- [x] Documentation was added or is not required

### CODE REVIEW

**Comment by rohit-nayak-ps** [comment]
PR realizes 15% reduction 
```
$ git checkout main
$ git grep -i master | wc -l
5750
$ git checkout rn-inclusive-naming-2 
$ git grep -i master | wc -l
5012
```

File: go/vt/proto/vttest/vttest.pb.go [hunk 1]
```diff
@@ -139,7 +139,7 @@ type Keyspace struct {
 	ShardingColumnType string `protobuf:"bytes,4,opt,name=sharding_column_type,json=shardingColumnType,proto3" json:"sharding_column_type,omitempty"`
 	// redirects all traffic to another keyspace. If set, shards is ignored.
 	ServedFrom string `protobuf:"bytes,5,opt,name=served_from,json=servedFrom,proto3" json:"served_from,omitempty"`
-	// number of replica tablets to instantiate. This includes the primary tablet.
+	// number of replica tablets to instantiate. This includes the master tablet.
```
Comment (deepthi) [inline go/vt/proto/vttest/vttest.pb.go]: this should be changed back to `primary`. maybe the source proto needs to be edited.

File: go/vt/proto/tabletmanagerservice/tabletmanagerservice_grpc.pb.go [hunk 2]
```diff
@@ -47,13 +47,13 @@ type TabletManagerClient interface {
 	ExecuteFetchAsApp(ctx context.Context, in *tabletmanagerdata.ExecuteFetchAsAppRequest, opts ...grpc.CallOption) (*tabletmanagerdata.ExecuteFetchAsAppResponse, error)
 	// ReplicationStatus returns the current replication status.
 	ReplicationStatus(ctx context.Context, in *tabletmanagerdata.ReplicationStatusRequest, opts ...grpc.CallOption) (*tabletmanagerdata.ReplicationStatusResponse, error)
-	// MasterStatus returns the current primary status.
```
Comment (deepthi) [inline go/vt/proto/tabletmanagerservice/tabletmanagerservice_grpc.pb.go]: Can you review all the changes in this file? Seems to be another case where the source proto needs to change.

File: go/vt/proto/tabletmanagerdata/tabletmanagerdata.pb.go [hunk 3]
```diff
@@ -3752,11 +3752,11 @@ type DemotePrimaryResponse struct {
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
-	// Position is deprecated, and is a string representation of a demoted primaries executed position.
```
Comment (deepthi) [inline go/vt/proto/tabletmanagerdata/tabletmanagerdata.pb.go]: One more proto file to fix.

**Review by deepthi — COMMENTED** [review]
Looks like some of the proto files need to be fixed further. `make proto` overwrote some of the manual edits.
If we are running `make proto` we should run `make proto grpcvtctldclient vtadmin_web_proto_types` to keep vtadmin in sync.

**Review by deepthi — APPROVED** [review]
:sweat_smile:

**Pull request merged**